content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -353,6 +353,36 @@ spec:
 {{< /tabs >}}
 
 
+### Version removal
```

Comment:
I noticed that at the top of the page in the Overview section there already are some steps regarding version removal. It does include setting served to false first, and also links to upgrading existing objects to a new stored version.

Maybe the existing steps are sufficient?

Reply:
Hmm, I see those steps in the Overview section.  I'm fine either way.